src/test/kotlin/se/uulm/snowballr/backend/validation/ProjectMemberValidatorTest.kt

```diff
@@ -0,0 +1,43 @@
+package se.uulm.snowballr.backend.validation
+
+import `in`.rcard.assertj.arrowcore.EitherAssert
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import se.uulm.snowballr.backend.model.InvalidId
+import snowballr.ProjectOuterClass.Project.Member.Remove
+import java.util.UUID
+
+class ProjectMemberValidatorTest {
+    @Nested
+    inner class RemoveRequest {
+        private val validCreateRequestBuilder: Remove.Builder =
```

Comment:
```suggestion
        private val validRemoveRequestBuilder: Remove.Builder =
```
Please be cautious when copy pasting.

Reply:
Sorry. Good catch, though.